docs/impact/index.md

```diff
@@ -1,48 +1,46 @@
+- Open software conversion and analytics tools, and global infrastructure for
+collaborating on emerging standards in neuroscience 
+[Poldrack, et al, 2024](https://direct.mit.edu/imag/article/doi/10.1162/imag_a_00103/119672/The-past-present-and-future-of-the-brain-imaging).
+- ~30,000 annual website visits from a very large community of neuroscience researchers 
+actively consulting the BIDS specifications (April 2023-24).
+- Over 300 BIDS contributors currently supporting and maintaining BIDS community 
+resources and tools.
+- Open working processes through online collaborative tools including GitHub, 
+GitHub Pages, ReadTheDocs, Google Docs, and OSF.
+  
+The BIDS core steering and maintainers groups work with the community to optimize open 
+adherence to FAIR principles (Findable, Accessible, Interoperable, and Reusable, 
+[Wilkinson, et al, 2016](https://www.nature.com/articles/sdata201618)) and to actively 
+lower barriers to adoption.
 
-This document intends to convey how the BIDS Steering and Maintainers Group can
-support your proposed BIDS grant. In an effort to express the impact of BIDS in
-the brain imaging community, we have shared several of our traffic metrics.
-
-## How the BIDS team can help you
-
-If you are in the process of putting together a grant, **please email/message
-the pillar lead** that is most closely associated with your proposed grant so we
-may help support this. Our organization is structured into 3 pillars: standard,
-tools, and collaboration.
-
-The respective leads are:
-
--   Stefan Appelhoff ([stefan.appelhoff@mailbox.org](mailto:stefan.appelhoff@mailbox.org)),
--   Chris Markiewicz ([markiewicz@stanford.edu](mailto:markiewicz@stanford.edu))
-
-Our range of support covers activities such as: meeting with the Steering and
-Maintainers Groups to assisting with connecting you with other BIDS grant
-writers or related initiatives to receiving a letter of support from the
-Steering Group.
-
-Regarding **requesting a letter of support** - please submit a drafted letter of
-support to the collaboration lead (???) so we may review internally.
-
-Please include how you plan on giving back to the BIDS community. The primary
-mechanism is to support a member of your team to become a BIDS Maintainer.
-Letters of support are approved by the Steering Group. This process may take 2-4
-weeks to complete.
-
-We kindly ask you to please **share your grant online** - successful or not.
-This will signal to the rest of the community what avenues we have pursued and
-what our fellow colleagues are planning on doing next. These can be grants that
-directly extend our support into a new domain, grants that help BIDS (for example
-OpenNeuro), or unsuccessful grant applications.
-
-For example, please find our [NIH-R24 Brain Initiative BIDS-Derivatives grant](https://osf.io/c3dgx/).
-
-A listing of the previous grants can be found [here](../collaboration/acknowledgments.md)
-
-## dashboards
+## Measuring BIDS Impact
```

Comment:
This looks much better on the render than it does here, I think stars could table could be a link/hidden, but that can be saved for another PR.